### PR TITLE
lb1

### PR DESCRIPTION
**Pull request by Marynok**

### CODE REVIEW

File: EF/DeliveryService/DeliveryService/Models/BaseModel/Model.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DeliveryService.Models.BaseModel
+{
+     public abstract class Model
```
Comment (yevhenbilushko) [inline EF/DeliveryService/DeliveryService/Models/BaseModel/Model.cs]: Model - not clear name
maybe "BaseModel"

File: EF/DeliveryService/DeliveryService/Models/BaseModel/Model.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+     { 
+        public int Id { get; set; }
+        public Model(){}
+
+        public Model(int id)
+        {
+            Id = id;
```
Comment (yevhenbilushko) [inline EF/DeliveryService/DeliveryService/Models/BaseModel/Model.cs]: no need methods

File: EF/DeliveryService/DeliveryService/Models/Address.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
+﻿using DeliveryService.Models.BaseModel;
+
+namespace DeliveryService.Models
+{
+    public class Address : Model
```
Comment (yevhenbilushko) [inline EF/DeliveryService/DeliveryService/Models/Address.cs]: please remove all methods from models
usually it is about data structure 
not about behavior 
so no need methods

File: EF/DeliveryService/DeliveryService/Models/Basket.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
+﻿using DeliveryService.Models.BaseModel;
+using System.Collections.Generic;
+
+
+namespace DeliveryService.Models
+{
+    public class Basket : Model
+    {
+        public Client Client { get; set; }
```
Comment (yevhenbilushko) [inline EF/DeliveryService/DeliveryService/Models/Basket.cs:8]: add ClientId

File: EF/DeliveryService/DeliveryService/Models/Delivery.cs [hunk 5]
```diff
@@ -0,0 +1,34 @@
+﻿using DeliveryService.Models.BaseModel;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DeliveryService.Models
+{
+    class Delivery: Model
```
Comment (yevhenbilushko) [inline EF/DeliveryService/DeliveryService/Models/Delivery.cs]: addd public

File: EF/DeliveryService/DeliveryService/Models/Delivery.cs [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        Order Order { get; set; }
```
Comment (yevhenbilushko) [inline EF/DeliveryService/DeliveryService/Models/Delivery.cs]: add public for all properties

File: EF/DeliveryService/DeliveryService/Models/Manufacturer.cs [hunk 7]
```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using DeliveryService.Models.BaseModel;
+
```
Comment (yevhenbilushko) [inline EF/DeliveryService/DeliveryService/Models/Manufacturer.cs]: remove empty lines

File: EF/DeliveryService/DeliveryService/Models/Manufacturer.cs [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+
+namespace DeliveryService.Models
+{
+    public class Manufacturer : Model
+    {
+        public string Name { get; set; }
+        public Address Address { get; set; }
```
Comment (yevhenbilushko) [inline EF/DeliveryService/DeliveryService/Models/Manufacturer.cs:10]: add AddressId

File: EF/DeliveryService/DeliveryService/Models/Food.cs [hunk 9]
```diff
@@ -0,0 +1,35 @@
+﻿using DeliveryService.Models.BaseModel;
+using System;
+
+
+namespace DeliveryService.Models
+{
+    public class Food : Model
+    {
+        public string Name { get; set; }
```
Comment (yevhenbilushko) [inline EF/DeliveryService/DeliveryService/Models/Food.cs:8]: please add ManufactureId

File: EF/DeliveryService/DeliveryService/Models/Basket.cs (hunk 4 above)
Comment (Marynok) [reply EF/DeliveryService/DeliveryService/Models/Basket.cs:8]: In this task has not used entity framework yet , can I  leave only a link to the object ?

Comment (yevhenbilushko) [reply EF/DeliveryService/DeliveryService/Models/Basket.cs:8]: yeah

**Review by yevhenbilushko — APPROVED** [review]

**Pull request merged**